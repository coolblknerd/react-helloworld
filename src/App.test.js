import React from 'react';
import App from './App';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

// Shallow rendering is used for isolated unit tests
describe('App', () => {
  it('should render correctly', () => {
    const output = shallow(
      <App />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });

  it('should render button that says "Click Me"', () => {
    const output = shallow(
      <App />
    );
    expect(output.find('button').text()).toEqual('Click Me');
  });

  // Really want to know how to test event
});

