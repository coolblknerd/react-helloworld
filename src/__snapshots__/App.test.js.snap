// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="button__container"
>
  <button
    className="button"
    onClick={[Function]}
  >
    Click Me
  </button>
  <p />
</div>
`;
